lib/Db/FeedMapperV2.php

```diff
@@ -158,20 +160,33 @@ public function findAllFromFolder(?int $id): array
      * @param string   $userId
      * @param int      $id
      * @param int|null $maxItemID
+     *
+     * @throws \OCP\DB\Exception
+     *
+     * @TODO Update for NC 21
      */
     public function read(string $userId, int $id, ?int $maxItemID = null): void
     {
-        $builder = $this->db->getQueryBuilder();
-        $builder->update(ItemMapperV2::TABLE_NAME, 'items')
+        $idBuilder = $this->db->getQueryBuilder();
+        $idBuilder->select('id')
```

Comment:
`1052: Column 'id' in field list is ambiguous`
```suggestion
        $idBuilder->select('items.id')
```